docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -75,19 +86,19 @@ For further information on distance metrics, see the <<available-metrics, Availa
 |`N/A`
 
 |max-degree
-|Used to calculate the maximum number of neighbors per node. The calculation used is max-degree * 2
-|Required
-|`16`
+|Maximum number of neighbors per node. Note that the meaning of this parameter differs from version 5.5.
+|Optional
```

Comment:
Is it accurate to write that this parameter is optional, given that it cannot be set to null?

Reply:
it has a default value, so it does not have to be configured explicitly. It cannot be set explicitly to null in Java, it should be similar in XML and YAML but I did not test what would happen.

Reply:
I define "optional" as "nullable or has reasonable default" (similar to https://docs.hazelcast.com/hazelcast/5.5/migrate/dmt-command-reference). If we want to inform precisely about nullability, it should be separate column. But IMO for the users the most import information is if they have to pass it or not.